packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -248,8 +250,8 @@ class ViewSettingsDialog extends UI5Element {
 			  sortOrderSelected = this._sortOrder.getSelectedItems(),
 			  sortBySelected = this._sortBy.getSelectedItems();
 
-		settings.sortOrder = sortOrderSelected.length ? sortOrderSelected[0] : undefined;
-		settings.sortBy = sortBySelected.length ? sortBySelected[0] : undefined;
+		settings.sortOrder = sortOrderSelected.length ? sortOrderSelected[0] : { innerText: "" };
```

Comment:
why do you change this?

Reply:
I figured that this._confirmedSettings.sortBy.innerText could always return value, the default could be empty string.
Compared to always check if this._confirmedSettings.sortBy exists and isn't undefined.
The same applied for this._confirmedSettings.sortOrder